crates/ark/src/treesitter.rs

```diff
@@ -460,6 +460,40 @@ pub(crate) fn node_is_call(node: &Node, name: &str, contents: &ropey::Rope) -> b
     fun == name
 }
 
+/// This function takes a [Node] that you suspect might be in a call argument position
+/// and walks up the tree, looking for the containing call node
+///
+/// If the [Node] is not a call argument, this returns `None`.
+///
+/// This works for `Call`, `Subset`, and `Subset2`, which all share the same tree-sitter
+/// structure.
+pub(crate) fn node_find_containing_call<'tree>(node: &Node<'tree>) -> Option<Node<'tree>> {
```

Comment:
I think the original name `node_find_parent_call()` was clearer?
Or were you worried "parent" implies immediate parent which is not technically the case? In that case, "ancestor" seems more conventional than "containing"?

Reply:
`node_find_parent_call()` is fine with me

Reply:
ok, done